dipy/tracking/tests/test_fbc.py

```diff
@@ -0,0 +1,49 @@
+from dipy.denoise.enhancement_kernel import EnhancementKernel
```

Comment:
EnhancementKernel 
-  __init
- create_lookup_table
- estimate_kernel_size
- k2
- coordinate_map
- kernel
- euler_angles
- R
- get_lookup_table
- get_orientations
- get_sphere (not tested here, but tested in test_kernel.py)

FBCMeasures
- __init
- get_points_rfbc_thresholded
- compute
- compute_rfbc
- min_moving_average

All functions are tested